extensions/oplogPopulator/OplogPopulator.js

```diff
@@ -77,30 +127,262 @@ class OplogPopulator {
         return config;
     }
 
+    /**
+     * Get buckets that have at least one extension active
+     * @returns {string[]} list of buckets to listen to
+     * @throws {InternalError}
+     */
+    async _getBackbeatEnabledBuckets() {
+        const filter = {
+            $or: [],
+        };
+        // setup filters
+        this._activeExtensions.forEach(extName => {
+            const field = constants.extensionNameToBucketMetadataConfigField[extName];
```

Comment:
`extensionNameToBucketMetadataConfigField` : may want to try and find a better name...